src/createQueryGroup.ts

```diff
@@ -51,6 +52,8 @@ export function createQueryGroup<
     executing.value = false
     
     setNextExecution()
+
+    return response.value
```

Comment:
This doesn't handle errors, we'll probably want to rework this a bit so that awaiting the execute method actually throws an error.